parameterized-utils.cabal

```diff
@@ -36,14 +36,15 @@ source-repository head
 
 library
   build-depends: base >= 4.10 && < 5
-               , th-abstraction >=0.3  && <0.4
```

Comment:
Can you revert the whitespace changes? Tools like `git blame` work better if we change whitespace less often.